hooks/pre/10-reset_data.rb

```diff
@@ -2,22 +2,22 @@
 require 'English'
 
 def reset
-  if foreman_installed?
+  if foreman_maintain_present?
```

Comment:
I'll be more verbose: previously this only worked if `foreman-rake` was present. That's only provided by the Foreman RPM. However, this can clear out Pulp and Pulpcore as well which in the future will be present on hosts without Foreman as well. This entire check is now no longer needed.

Reply:
OK great. I changed the check to foreman-maintain because I believe it should be present on smart proxies while foreman-rake is not. But I wasn't aware that foreman-maintain doesn't exist on Debian.

With that said, even without this check, we are still relying on foreman-maintain -- it's used in the definition of stop_services which runs unconditionally, so if that's not present on Debian this will still fail.